src/App.css

```diff
@@ -2,4 +2,20 @@
 
 * {
     font-family: 'Montserrat', sans-serif;
+}
+
+.logo {
```

Comment:
These three classes .logo, .nav-link and .nav-link:nth-child should be moved to Header.js as well